docs/cloud_orchestrator.md

```diff
@@ -0,0 +1,17 @@
+# Cloud Orchestrator
+
+This page describes about `Cloud Orchestrator`.
+
+## What's Cloud Orchestrator?
+
+Cloud Orchestrator is a web service that hosting virtual machines or containers
+which are able to run Cuttlefish instance on top of it.
+It can create, list, and delete host instances, and provides the way to access
+hosts to launch Cuttlefish instances on top of the host instance via Host
+Orchestrator.
+
+## Use Cloud Orchestrator
+
+We're currently providing using Cloud Orchestrator with Docker instances as
+hosts. Please read [docker.md](docker.md) to follow.
+<!-- TODO(ser-io): Write how to use CO for GCP. -->
```

Comment:
@ser-io I'm not sure if the usage about CO with GCP should be written here. Unless, I'll remove the TODO comment.